drafttopic/utilities/tests/test_fetch_wikiprojects.py

```diff
@@ -0,0 +1,113 @@
+from nose.tools import eq_
+from ..fetch_wikiprojects import WikiProjectsParser
+from ..fetch_wikiprojects import WPDPage
+import logging
+import json
+import os
+import pdb
+
+logger = logging.getLogger( __name__ )
+logging.basicConfig(level=logging.INFO)
+
+excluded_keys = ['topics', 'root_url', 'index', 'name', 'url']
+
+def test_WikiProjectsFromTable():
```

Comment:
functions should be snake case https://www.python.org/dev/peps/pep-0008/#function-names

Reply:
fixed.